### DISCUSSION TITLE
Plane Integration

### DISCUSSION THREAD

**Discussion by sats268842** [outline/outline#6718] [Ideas]
We would like to request the development of an integration between Plane.so and Outline

Link: https://plane.so

**Comment by vcscroll** [comment]
This makes a lot of sense, @sats268842, what kind of integration you are talking about?

What would you expect to integrated first?

**Reply by sats268842** [reply]
@vcscroll  The way Jira and Confluence function together, such as attaching plane issues to outline documents.

**Reply by CatrinelF** [reply]
Plus one to this integration - mostly automatic status update on reports + roadmap view would be great as a start

**Comment by tommoor** [comment]
The GitHub integration is now completed and could be used as a model here. Right now this will show information from GitHub in a "hover card", though the plan is to be able to embed this in a document once the editor supports it. Any integrations following the same unfurl pattern that the GitHub one has defined would get this behavior for free once it's implemented in the coming months.
